dummy_robot/dummy_robot_bringup/launch/dummy_robot_bringup.py

```diff
@@ -12,36 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-
 from ament_index_python.packages import get_package_share_directory
-from ros2run.api import get_executable_path
-
-file_path = os.path.dirname(os.path.realpath(__file__))
-
-
-def launch(launch_descriptor, argv):
-    ld = launch_descriptor
-
-    package = 'dummy_map_server'
-    ld.add_process(
-        cmd=[get_executable_path(package_name=package, executable_name='dummy_map_server')],
-    )
-
-    package = 'robot_state_publisher'
-    ld.add_process(
-        cmd=[
-            get_executable_path(package_name=package, executable_name='robot_state_publisher'),
-            os.path.join(
-                get_package_share_directory('dummy_robot_bringup'), 'launch', 'single_rrbot.urdf')
-        ],
-    )
-
-    package = 'dummy_sensors'
-    ld.add_process(
-        cmd=[get_executable_path(package_name=package, executable_name='dummy_laser')],
-    )
-
-    ld.add_process(
-        cmd=[get_executable_path(package_name=package, executable_name='dummy_joint_states')],
-    )
+from launch import LaunchDescription
+import launch.actions
+from launch_ros.substitutions import ExecutableInPackage
+
+
+def generate_launch_description():
+    return LaunchDescription([
+        launch.actions.ExecuteProcess(
+            cmd=[ExecutableInPackage(package='dummy_map_server', executable_name='dummy_map_server')],
+            output='screen'),
+        launch.actions.ExecuteProcess(
+            cmd=[
+                ExecutableInPackage(package='robot_state_publisher', executable_name='robot_state_publisher'),
+                os.path.join(
```

Comment:
Can you put a TODO comment here to use a substitution to find this file rather than using `ament_index_python` directly. I haven't implemented the substitution yet, but it is planned.